### DISCUSSION TITLE
Usync migrations should do full export post finished migration

### DISCUSSION THREAD

**Discussion by bielu** [Jumoo/uSyncMigrations#234] [Ideas]
On one of projects I noticed weirdness about usync files post migration, and it was caused by small but significant different between usync files which cause having 2 same usync files with different ids but for same data type and they were breaking solution (not sure what happened there).
i think it will be worth to run full clean export of setting spost migration

**Comment by KevinJump** [comment]
Yes - but we would have to make this optional somehow, because people do run conversions (where we take v9 folder and convert) and they do this multiple times, if we performed a full export every time, they couldn't run it again. 🤔

**Reply by bielu** [reply]
We can make flag in app settings to switch on/off or use migration plan to do it.
